apps/billing/payments/zibal_client.py

```
@@ -88,52 +90,60 @@ def __init__(self):
+                    'endpoint': endpoint,
+                    'result': result.get('result'),
+                    'gateway_message': result.get('message', ''),
+                },
+            )
+
             return result
-            
+
         except requests.exceptions.Timeout as e:
-            logger.error(f"Zibal timeout on {endpoint}", exc_info=True)
+            logger.error("Zibal timeout on %s", endpoint, exc_info=True)
```

Ruff (0.14.0)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)